### DISCUSSION TITLE
为什么发送还是跳转浏览器啊

### DISCUSSION THREAD

**Discussion by SDCOM-0415** [share121/inter-knot#4881] [其他]
为什么发送还是跳转浏览器啊

**Comment by IXfromThorne** [comment]
不造哇（吃手手）

**Comment by scrooge-0** [comment]
这个是基于github写的程序，相当于借用了github的服务器来聊天，不然我们的开发者是没办法弄这么多钱来搞服务器的，详情参见有人弄的龙族网站，采用了自己的服务器，但是非常卡，而且不能显示图片
